android/src/main/java/com/reactnativestripesdk/CollectBankAccountLauncherFragment.kt

```diff
@@ -32,6 +34,18 @@ class CollectBankAccountLauncherFragment(
 ) : Fragment() {
   private lateinit var collectBankAccountLauncher: CollectBankAccountLauncher
 
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    val stripeSdkModule: StripeSdkModule? = context.getNativeModule(StripeSdkModule::class.java)
+    if (stripeSdkModule != null && stripeSdkModule.eventListenerCount > 0) {
+      FinancialConnections.setEventListener { event ->
+        val params = mapFromFinancialConnectionsEvent(event)
+        stripeSdkModule.sendEvent(context, "onFinancialConnectionsEvent", params)
+      }
+    }
```

Comment:
This is a “best guess” approach, since I don’t see a way for us to reliably track if a listener is registered for `onFinancialConnectionsEvent`.